torch/csrc/autograd/functions/convolution_attr.h

```diff
@@ -0,0 +1,105 @@
+#pragma once
+
+#include <Python.h>
+#include "convolution.h"
+#include "torch/csrc/autograd/python_cpp_function_attr.h"
+
+
+namespace torch { namespace autograd {
+
+namespace attributes {
+
+template<class T>
+PyObject* conv_stride(THPCppFunction* self, PyObject* hook)
```

Comment:
Also, once the definitions are smaller they don't need to be in an additional file.